code/modules/client/preference_setup/loadout/loadout_ears.dm

```diff
@@ -11,4 +11,31 @@
 
 /datum/gear/ears/circuitry
 	display_name = "earwear, circuitry (empty)"
-	path = /obj/item/clothing/ears/circuitry
\ No newline at end of file
+	path = /obj/item/clothing/ears/circuitry
+	
+/datum/gear/ears/earrings
+	display_name = "earring selection"
+	description = "A selection of eye-catching earrings."
+	path = /obj/item/clothing/ears/earring
+
+ /datum/gear/ears/earrings/New()
+	..()
+	var/earrings = list()
+	earrings["stud, pearl"] = /obj/item/clothing/ears/earring/stud
```

Comment:
Travis complains about indentation:
![awk](https://puu.sh/EeGZb/611f4a2bc5.png)
and compilation:
![compile](https://puu.sh/EeGZo/cb4b03d574.png)

Reply:
This is actually a placeholder PR. It'll be done by Monday!